sphinxcontrib/autoprogram.py

```diff
@@ -140,7 +140,7 @@ def import_object(import_name):
                 mod = __import__("foo")
                 break
         else:
-            raise ImportError("No module named {}".format(module_name))
+            raise
```

Comment:
How about using [`raise ... from ...` syntax][0] instead?  We could use [`six.raise_from()` function][1] for Python 2 compatibility.

[0]: https://www.python.org/dev/peps/pep-3134/
[1]: https://six.readthedocs.io/#six.raise_from

Reply:
I guess the message is already there, and it uses the original `ModuleNotFoundError` (subclassing from `ImportError`) on py3. An average user probably wants to get the original trace under Python 2, too.